python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -118,7 +115,7 @@ def __init__(self,
                             format='[%(asctime)s] %(levelname)-8s %(message)s',
                             datefmt='%Y-%m-%d %H:%M:%S'
                             )
-        self.logger = logging.getLogger(__name__)
+        BaseRunner.__init__(self, logging.getLogger(__name__))
```

Comment:
super(TorchRunner).init?

Reply:
this is a typo, there is no need to init logger in baserunner(as logger is not used). removed now.